packages/gen-ai-hub/src/index.ts

```diff
@@ -6,3 +6,7 @@ export {
   OpenAiChatCompletionOutput
 } from './client/index.js';
 export { CustomRequestConfig, BaseLlmParameters } from './core/index.js';
+export {
+  getOrchestrationClient,
+  OrchestrationClientRequest
```

Comment:
Please use "GenAIHub" instead of "orchestration" in general, as the "orchestration" service is not one official service that has been approved.
We can use that for internal/non-official communication.
As public API, we have to avoid using it.

You can either rename it or add additional wrapper that points to the orchestration functions.